src/main/java/ch/njol/skript/conditions/CondIsInLove.java

```diff
@@ -0,0 +1,55 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Animals;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Entity In Love")
+@Description("Whether or not an animal is currently in a love state")
+@Examples({
+	"on right click on living entity:",
+	"\tif event-entity is in love:",
+	"\t\tsend \"&c&oOhhh, he's in love <3\" to player"
+})
+@Since("INSERT VERSION")
+public class CondIsInLove extends PropertyCondition<LivingEntity> {
+
+	static {
+		register(CondIsInLove.class, "in love", "livingentities");
```

Comment:
```suggestion
		register(CondIsInLove.class, "in love[[ing] state]", "livingentities");
```

Reply:
This provides an improper grammar `loveing` should be `loving` I'll make a commit for this soon hopefully to fix it